spec/presenters/planned_disbursement_xml_presenter_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+
+RSpec.describe PlannedDisbursementXmlPresenter do
+  let(:planned_disbursement) { build_stubbed(:planned_disbursement) }
+
+  describe "#period_start_date" do
+    it "returns a date formatted for IATI XML" do
+      planned_disbursement.period_start_date = "25 June 2020"
+      expect(described_class.new(planned_disbursement).period_start_date).to eq("2020-06-25")
+    end
+  end
+
+  describe "#period_end_date" do
+    it "returns a human readable date" do
+      planned_disbursement.period_end_date = "2020-10-20"
+      expect(described_class.new(planned_disbursement).period_end_date).to eq("2020-10-20")
+    end
+  end
+
+  describe "#value" do
```

Comment:
@lozette I also don't like this one - it doesn't actually do what it says - I stole it from another presenter spec, but I am concerned that if I do anything to the value I am introducing rounding errors to the monetary value? Do you have any thoughts on what we can do to output this as `x.00` - then we could apply it elsewhere?

Reply:
How about `ActionController::Base.helpers.number_to_currency(super, unit: "", delimiter: "")` - similar to the non-XML presenter but without the unit and the comma delimiter.